### DISCUSSION TITLE
What is the best practice for disabling the default Dialogic button sounds?

### DISCUSSION THREAD

**Discussion by aumusic** [dialogic-godot/dialogic#1783] [Q&A] [answered]
I didn't realize this until recently, probably due to my habit of having things playing in the background while working on Godot/Dialogic stuff, but I just realized earlier today that Godot seemingly has default sounds for when the cursor moves over or clicks a Dialogic choice button. The relevant script file for controlling this, or customizing it, seems to be the node_button_sound.gd file, but since I'm not seeing anything within the Dialogic interface pertaining to disabling the sounds outright, what is considered the 'best practice' for disabling these sounds?

Thus far, I've tested going into the node_button_sound.gd script and commenting out the relevant else arguments, which does seem to do the trick without generating errors, but might this eventually cause problems? I've also thought about replacing the audio files with silent audio files, which would have the benefit of not changing any of the existing code, but I'm also not exactly sure yet which audio files are being referenced (I did find some 'typing' wav files in one directory, but re-naming them didn't change anything. I haven't poked around any further on this one).

Are there 'better' ways of accomplishing the same goal?

(EDIT: since I *constantly* forget to do this, I should mention I'm in Godot 4.1.1 with Dialogic 2.0 Alpha 9)

Thanks in advance

**Comment by Jowan-Spooner** [comment, marked as answer]
Hey @aumusic, thanks for your thoruogh research before asking here!

Right now the best way of getting rid of them is to createa a custom layout scene (probably a copy of the default layout scene) and remove all set audio files on the DialogicNode_ButtonSound node.

However I will probably add audio settings for choice sounds to the default layout along with the fix for this request #1782.

Also (although I don't think it's of much relevance) 
- it is indeed using some of the typing sounds right now
- commenting out code is not a problem in itself, but any changes in the addons directory will be lost on a subsequent update

Hope this helps

**Comment by Jowan-Spooner** [comment]
I've just exposed choice button sound effects to the default layout on main so this will be in the next release.
